lib/DeployActions/AIODockerActions.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\AppEcosystemV2\DeployActions;
+
+use OCA\AppEcosystemV2\Db\DaemonConfig;
+use OCA\AppEcosystemV2\Service\DaemonConfigService;
+use OCP\IConfig;
+
+/**
+ * Class with utils methods for AIO setup
+ */
+class AIODockerActions {
+	public const AIO_DAEMON_CONFIG_NAME = 'docker_aio';
+	public const AIO_DOCKER_SOCKET_PROXY_HOST = 'nextcloud-aio-docker-socket-proxy:2375';
+	public const AIO_DAEMON_HOST = 'http://nextcloud-aio-apache:11000/index.php';
```

Comment:
Just a remark herr: it would probably better to point this at https://$NC_DOMAIN as port 11000 is not always choosen but the above should usually work always...

Reply:
Though probably not on your dev instance since you are running with self-signed cert...

Reply:
For production instances AIO always requires a valid cert for the chosen domain and always runs on the default port 443. that is why the above should work.

Reply:
Hmm good points. @bigcat88 What do you think?

Reply:
> For production instances AIO always requires a valid cert for the chosen domain and always runs on the default port 443. that is why the above should work.

So locally for testing we could always create our own daemon and user apache proxy for ExApps to communicate to nextcloud, instead of this automatically created. Thanks for pointing on this)